sematic/examples/testing_pipeline/__main__.py

```diff
@@ -175,6 +181,11 @@ def _parse_args() -> argparse.Namespace:
         dest="exit_code",
         help=EXIT_HELP,
     )
+    parser.add_argument(
+        "--ray-cluster-address",
```

Comment:
Are users going to have to have this as well in their pipeline parametrization?

Reply:
No, this is a temporary measure until we have an `ExternalResource` that can spin up a cluster as a part of the pipeline execution